### ISSUE TITLE
✨ [Feature] Cors middleware can be disable

### ISSUE THREAD

**Issue by devhaozi** [goravel/goravel#446]
### Before feedback (在反馈之前)

- [X] There are no features that I want to submit in Issues (当前 Issues 中没有我将要提交的新功能)

### Describe Feature (描述功能)

Default setting cors `allowed_origins` to `*` is dangerous, suggest default disable cors middleware and users can choose whether to enable it.

**Comment by devhaozi** [comment]
Temporary solution: set a cors paths that will never be used.

**Comment by xx1196** [comment]
hi, i have this config package config

import (
	"github.com/goravel/framework/facades"
)

func init() {
	config := facades.Config()
	config.Add("cors", map[string]any{
		// Cross-Origin Resource Sharing (CORS) Configuration
		//
		// Here you may configure your settings for cross-origin resource sharing
		// or "CORS". This determines what cross-origin operations may execute
		// in web browsers. You are free to adjust these settings as needed.
		//
		// To learn more: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
		"paths":                []string{"*"},
		"allowed_methods":      []string{"*"},
		"allowed_origins":      []string{"*"},
		"allowed_headers":      []string{"*"},
		"exposed_headers":      []string{""},
		"max_age":              0,
		"supports_credentials": false,
	})
} and my frontend when make a request this CORS is failed, how i can fix this?

**Comment by hwbrzzl** [comment]
@xx1196 Do you want to enable CORS or not?

**Comment by xx1196** [comment]
Yes, but I put the domain in the allows domain but I have the same error with * or my domain I don't know who I can fix it :(

**Comment by hwbrzzl** [comment]
What's your latest CORS configuration now?

**Comment by xx1196** [comment]
At this moment it is the default one, and it does not work

**Comment by xx1196** [comment]
Is there something that can be done about it?

**Comment by hwbrzzl** [comment]
I  use the code below in a third website to call the local endpoint, the cors works well. Notice, if `allowed_origins` is empty, it means that all domains are allowed, need to set a specific value.

```
fetch("http://127.0.0.1:3000/lang", {
    method: "GET"
})
.then(response => {
    if (!response.ok) {
        throw new Error("error: " + response.status);
    }
    return response.json();
})
.then(data => console.log("success:", data))
.catch(error => console.log("error:", error));
```

**Comment by xx1196** [comment]
first of all need use vhost like my-backend.test and the request send from my-frontend.test
second point i can resolve using facades.Route().Get("/preferencial_cares", preferencialCareController.Index) instead of facades.Route().Prefix("group").Group(func(router route.Router)

**Comment by hwbrzzl** [comment]
This feature should be good, welcome back when needed.

**Comment by hwbrzzl** [comment]
Solution: Set `cors.path: []string{}`, updated the default value on goravel/goravel.
